src/install-deps.js

```diff
@@ -0,0 +1,37 @@
+const path = require("path");
+const glob = require("glob");
+const cp = require("child_process");
+const parallelLimit = require("run-parallel-limit");
+const series = require("run-series");
+
+module.exports = function installFunctionDeps(src) {
+  const fns = functionGlobber(src)
+  return new Promise((resolve, reject) => {
+    if (fns.length === 0) return resolve()
+
+    const jobs = fns.map(fnPath => {
```

Comment:
worth filtering something here in case some function folders do not have a package.json?

Reply:
The globber should only select folders with package.json if I understand correctly:

https://github.com/netlify/zip-it-and-ship-it/pull/34/files#diff-f1580f02b18be88d05da772c352b9bbaR27

Reply:
yeah it ignores folders without `package.json`